dask/array/core.py

```diff
@@ -2379,11 +2385,13 @@ def to_hdf5(filename, *args, **kwargs):
     else:
         raise ValueError("Please provide {'/data/path': array} dictionary")
 
+    chunks = kwargs.pop('chunks', None)
+
     import h5py
     with h5py.File(filename) as f:
         dsets = [f.require_dataset(dp, shape=x.shape, dtype=x.dtype,
-                                        chunks=tuple([c[0] for c in x.chunks]),
-                                        **kwargs)
+                           chunks=chunks or tuple([c[0] for c in x.chunks]),
```

Comment:
It's useful to write unchunked datasets -- which is normally done by default with h5py. Maybe set the default chunks to `True` and let `chunks=None` indicate no chunks?